src/webui/webui.cpp

```diff
@@ -65,11 +66,14 @@ void WebUI::init()
 
 #ifndef QT_NO_OPENSSL
         if (pref->isWebUiHttpsEnabled()) {
-            QList<QSslCertificate> certs = QSslCertificate::fromData(pref->getWebUiHttpsCertificate());
-            QSslKey key;
-            key = QSslKey(pref->getWebUiHttpsKey(), QSsl::Rsa);
-            bool certsIsNull = std::any_of(certs.begin(), certs.end(), [](QSslCertificate c) { return c.isNull(); });
-            if (!certsIsNull && !certs.empty() && !key.isNull())
+            QSslKey key(pref->getWebUiHttpsKey(), QSsl::Rsa);
```

Comment:
It would be better to store key (I mean `getWebUiHttpsKey()` value) since Preferences getter is expensive.